docs/site/Loopback-component-authorization.md

```diff
@@ -0,0 +1,346 @@
+---
+lang: en
+title: 'Authorization'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Loopback-component-authorization.html
+---
+
+## Overview
+
+In every web application, we need to have a way to identify access rights of a
+user for any resource, which is known as **Authorization**. This is a
+minimalistic guide for creating such an implementation using Loopback component.
+This can be part of your main REST Application project or can be created as a
+Loopback extension for reuse in multiple projects. Latter is the better option
+for obvious reasons - reusability.
```

Comment:
We need to make it clear that `authorization` will be shipped with LoopBack 4 as an out-of-box feature. This guide is interim before the component is released.

Reply:
Added this as a NOTE at the top.